### ISSUE TITLE
Vehicle purchasing screen: Sliders offset on new game, correcting on mouse hover

### ISSUE THREAD

**Issue by FilmBoy84** [OpenApoc/OpenApoc#1180]
Starting a new game on `OpenApoc-x64-20230416.2-5-g1499fd4c`

1. I navigate to the vehicle purchising screen that initially looks like this
![image](https://user-images.githubusercontent.com/33913088/233308377-622a0ab2-5f2b-4309-a5e4-e4d58f963447.png)

2. I hover the mouse over the first item, and it corrects itself
![image](https://user-images.githubusercontent.com/33913088/233308471-6232d34b-d8a9-42be-910d-49da0fc5e7dd.png)

3. Continuing to hover the mouse over other items fixes them too
![image](https://user-images.githubusercontent.com/33913088/233308564-378e8275-2e93-4cf7-aed2-544c4eeee6c0.png)

**Expected behavior**
The result after the mouse hover should be what is initially displayed from the moment the screen is loaded

**Comment by Kurtsley** [comment]
Interesting, I never play on this resolution so never spotted this. Was concerned I broke something with that listbox update but it seems to exist on versions before that change.

**Comment by FilmBoy84** [comment]
Yeah seems to have been an issue for a while using some of my older builds

I think it may be related to the way the form is loaded - the actual values of the items don't seem to get a visual slider representation until the cursor hover forces an "update"

**Comment by Kurtsley** [comment]
This doesn't seem to be an issue anymore.
